Resources/Prototypes/_Goobstation/Entities/Clothing/Belt/belts.yml

```
@@ -232,3 +234,30 @@
         whitelist:
           tags:
           - Sidearm
+
+- type: entity
+  parent: [ClothingBeltStorageBase, ContentsExplosionResistanceBase, BaseCentcommContraband]
+  id: ClothingBeltMilitaryWebbingBSOFilled
+  name: blueshield chest rig
+  description: A set of tactical webbing worn by Central Command support units.
+  components:
+  - type: Sprite
+    sprite: Clothing/Belt/militarywebbing.rsi
+  - type: Clothing
+    sprite: Clothing/Belt/militarywebbing.rsi
+  - type: ExplosionResistance
+    damageCoefficient: 0.05 #goobstation
+  - type: StorageFill
+    contents:
+    - id: CombatInjector
```

```suggestion
  - type: ExplosionResistance
    damageCoefficient: 0.05
  - type: StorageFill
    contents:
    - id: CombatInjector
```
i noticed

waiter, waiter, there's a goobstation comment in my goobstation namespace soup